### ISSUE TITLE
RStudio randomly and inconsistently crashes

### ISSUE THREAD

**Issue by Benfeitas** [rstudio/reticulate#822]
I am using a conda environment to run a Rmd script that mixes R and python code through reticulate. I also have some python classes that I need to call, some of which use R code inside through `rpy2`. However, my RStudio is inconsistently crashing when I call these classes that run R code.

I was able to find the same behavior when simply using `rpy2` to call standard R functions. It all crashes when running the last `python` chunk below. This very often crashes, but not always:

> ```
> ---
> title: "Untitled"
> output: html_document
> ---
> ```
>```
>```{r Setup, echo = FALSE, warning=FALSE}
> library(reticulate)
> use_condaenv("my_env")```
> ```
> 
> ```
> ```{python, warnings=FALSE, message=FALSE, results='hide'}  
> ## rpy2
> import pandas as pd
> import rpy2
> from rpy2.robjects import r, pandas2ri
> pandas2ri.activate()
> import rpy2.robjects as ro
> from rpy2.rinterface import RRuntimeWarning```
> ```
> 
> ```
> ```{python}
> ##### often (but not always) this crashes R studio
> ro.r('''
>   library(limma)
>   ''')```
> ```

Calling `library(limma)` in an independent R script, or in a R chunk, does not crash anything.

This is my result of `py_config()`:
```
python:         /path/to/my_env/bin/python
libpython:      /path/to/my_env/lib/libpython3.7m.dylib
pythonhome:     /path/to/my_env:/path/to/my_env
version:        3.7.1 | packaged by conda-forge | (default, Nov 13 2018, 10:30:07)  [Clang 4.0.1 (tags/RELEASE_401/final)]
numpy:          /path/to/my_env/lib/python3.7/site-packages/numpy
numpy_version:  1.19.1
```

This is my `sessionInfo()`:
```
R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: macOS  10.15.5

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /path/to/my_env/lib/R/lib/libRblas.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] tools     stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] reticulate_1.15 limma_3.40.0   

loaded via a namespace (and not attached):
[1] compiler_3.5.1  Matrix_1.2-18   yaml_2.2.1      Rcpp_1.0.4.6    grid_3.5.1      knitr_1.28      jsonlite_1.6.1  xfun_0.13       lattice_0.20-41
```

I have a hard time reproducing this issue: often, creating a new conda environment **does not** help, and sometimes just re-running the code chunks a few times ends up working. I'm really at a loss on how to debug this, as I see no error messages. Any help is appreciated

**Comment by Benfeitas** [comment]
While I don't see any error messages, I'm wondering if this is the same problem as https://github.com/rstudio/reticulate/issues/208 given that it is reticulate + rpy2. I'm using `rpy2 3.1.0`, and `rstudio 1.2.502`.

**Comment by t-kalinowski** [comment]
Fixed via #1188
